### PR TITLE
Adding references to the MS Learn modules

### PR DESCRIPTION
**Pull request by geduardo**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/index.yml](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/a867a98f707ce0a637be2a779ae7493c24adf0be/articles/index.yml) | 87 | 54 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b4284bc5-c316-4f4a-8ade-445c35260ce8_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/welcome.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/a867a98f707ce0a637be2a779ae7493c24adf0be/articles/welcome.md) | 85 | 73 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8ecb06ff-8269-49b6-8371-00d5f541cd6b_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: articles/index.yml [hunk 1]
```diff
@@ -31,6 +31,8 @@ landingContent:
             url: overview/what-is-qsharp-and-qdk.md
           - text: Linear algebra for quantum computing
             url: overview/algebra-for-quantum-computing.md
+          - text: Microsoft Learn modules for quantum computing
+            url: https://docs.microsoft.com/learn/browse/?term=quantum
```
Comment (KittyYeungQ) [inline articles/index.yml:35]:
```suggestion
            url: https://docs.microsoft.com/en-us/users/buildcollections2020-6557/collections/1o2iogrmn8x4r
```
This is the link we use at Build.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/index.yml](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/d6b72d5c980835ac41ccb1c01f999c033cd0d118/articles/index.yml) | 87 | 54 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f6d4a070-8560-4867-9786-a2ab22ffe2bf_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/welcome.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/d6b72d5c980835ac41ccb1c01f999c033cd0d118/articles/welcome.md) | 85 | 73 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/30585b11-b1da-44b3-a8e5-42ddf6e1f369_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/index.yml](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/87e5d30281170300c11f959b82f6499fbcf736e6/articles/index.yml) | 87 | 54 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ca03a7d4-5b27-4836-afa2-7f74c673de0e_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/welcome.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/87e5d30281170300c11f959b82f6499fbcf736e6/articles/welcome.md) | 85 | 73 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f5983e0d-f4e2-494b-b488-5416d81e70d7_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Pull request merged**